mlflow/entities/model_registry/__init__.py

```diff
@@ -1,12 +1,14 @@
 from mlflow.entities.model_registry.registered_model import RegisteredModel
```

Comment:
Should I add docs to this PR or in a follow-up PR? It's already rather large and don't want to make it too big, but will follow best practices.

Reply:
I'd leave that for a follow up ;)